src/shared/cache/TreatmentMolecularDataCache.ts

```diff
@@ -1,8 +1,10 @@
 import LazyMobXCache, {AugmentedData} from "../lib/LazyMobXCache";
-import {TreatmentMolecularData, TreatmentDataFilterCriteria} from "../api/generated/CBioPortalAPIInternal";
-import client from "shared/api/cbioportalInternalClientInstance";
+import internalClient from "shared/api/cbioportalInternalClientInstance";
+import client from "shared/api/cbioportalClientInstance";
 import _ from "lodash";
 import {IDataQueryFilter} from "../lib/StoreUtils";
+import { SampleMolecularIdentifier, GenericAssayDataMultipleStudyFilter, GenericAssayData, GenericAssayDataFilter } from "shared/api/generated/CBioPortalAPI";
+import { fetchTreatmentData, TreatmentMolecularData } from "shared/lib/GenericAssayUtils/TreatmentUtils";
 
 export type TreatmentMolecularDataEnhanced = TreatmentMolecularData & {
```

Comment:
This functionality shoud not be restricted to the TreatmentMolecularData but part of GenericAssayData